### PR TITLE
Lotame id system domain update

### PR DESCRIPTION
**Pull request by markaconrad**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Define method on userId example page so the page loads
A few enhancements to the Lotame Panorama Id System User Id module

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: integrationExamples/gpt/userId_example.html [hunk 1]
```diff
@@ -71,10 +71,12 @@
                 t.async = false;
                 t.src = 'http://acdn.adnxs.com/cmp/cmp.bundle.js';
                 var tag = document.getElementsByTagName('head')[0];
-                tag.appendChild(t);
+                tag.appendChild(t);
```
Comment (jialong) [inline integrationExamples/gpt/userId_example.html:74]: Seems that extra space got added ?

File: integrationExamples/gpt/userId_example.html [hunk 2]
```diff
@@ -71,10 +71,12 @@
                 t.async = false;
                 t.src = 'http://acdn.adnxs.com/cmp/cmp.bundle.js';
                 var tag = document.getElementsByTagName('head')[0];
-                tag.appendChild(t);
+                tag.appendChild(t);                
             }
         })(window, document);
         // window.__cmp('showConsentTool');
+
+        function getHashedEmail() {}
```
Comment (noahpc) [inline integrationExamples/gpt/userId_example.html:79]: Will this file be included in the prebid pr or just in our repo?

File: modules/lotamePanoramaIdSystem.js [hunk 3]
```diff
@@ -14,10 +14,41 @@ const KEY_EXPIRY = `${KEY_ID}_expiry`;
...
+  const domainParts = fullDomain.split('.');
+  if (domainParts.length == 2) {
+    return fullDomain;
+  }
+  let rootDomain;
+  let continueSearching = true;
+  let startIndex = -2;
+  const TEST_COOKIE_NAME = 'lotame_domain_check';
+  const TEST_COOKIE_VALUE = 'writeable';
+  do {
```
Comment (noahpc) [inline modules/lotamePanoramaIdSystem.js:36]: Have we sanity checked this on a couple different browsers?

**Review by jialong — APPROVED** [review]

File: modules/lotamePanoramaIdSystem.js [hunk 4]
```diff
@@ -14,10 +14,41 @@ const KEY_EXPIRY = `${KEY_ID}_expiry`;
 const KEY_PROFILE = '_cc_id';
 const MODULE_NAME = 'lotamePanoramaId';
 const NINE_MONTHS_MS = 23328000 * 1000;
-const DAYS_TO_CACHE = 7;
-const DAY_MS = 60 * 60 * 24 * 1000;
 
 export const storage = getStorageManager(null, MODULE_NAME);
+const cookieDomain = findCookieDomain(String(document.domain));
```
Comment (noahpc) [inline modules/lotamePanoramaIdSystem.js:19]: Any tests that need to be added to the codebase?

**Review by noahpc — APPROVED** [review]

File: integrationExamples/gpt/userId_example.html (hunk 2 above)
Comment (markaconrad) [reply integrationExamples/gpt/userId_example.html:79]: This is the existing file in their repo that is broken. It's an example page but that method not being defined caused me a bit of confusion

File: modules/lotamePanoramaIdSystem.js (hunk 4 above)
Comment (markaconrad) [reply modules/lotamePanoramaIdSystem.js]: Yes, I found a way and added some

File: modules/lotamePanoramaIdSystem.js (hunk 3 above)
Comment (markaconrad) [reply modules/lotamePanoramaIdSystem.js]: Checked with firefox and then needed to refactor...

**Review by jialong — APPROVED** [review]

**Review by noahpc — APPROVED** [review]

**Pull request closed without merging**